worlds/civ_6/Locations.py

```diff
@@ -0,0 +1,193 @@
+from typing import List, Optional, Dict
+from BaseClasses import Location, LocationProgressType, Region
+
+from .Data import get_boosts_data, get_new_civic_prereqs_data, get_new_civics_data, get_new_tech_prereqs_data, get_new_techs_data
+
+from .Enum import CivVICheckType, EraType
+
+CIV_VI_AP_LOCATION_ID_BASE = 5041000
+
+# Locs that should have progression items (keypoint techs/civics, ~1 per era)
+PRIORITY_LOCATIONS = [
+    "TECH_ANCEINT_09",
```

Comment:
Is this really "ANCEINT" and not "ANCIENT"?
```suggestion
    "TECH_ANCIENT_09",
```